### ISSUE TITLE
Solar and wind power potential color scale overlaps carbon intensity scale

### ISSUE THREAD

**Issue by b2000699** [tmrowco/electricitymap-contrib#448]
Hi,

It would be good if someone could deactivate the carbon intensity coloring so to have a clear view on the solar power colors! For instance right now Greece is yellow-orange in carbon intensity and since high solar power potential is yellow, too, I cannot distinguish the solar power potential of Greece at all.

Therefore I propose to add a switch to be able to deactivate the carbon intensity coloring completely.
Cheers

**Comment by brunolajoie** [comment]
Interesting idea, but then if all country are grey we won't differentiate the one with and the one without data. What dya think @oli?

**Comment by b2000699** [comment]
I believe that someone should be able to deselect all layers and have the plain map only, just showing countries with no layer. So to be able to distinguish countries with data and those without, the basic question would be how to color these two.
In any case it should not be the same color as used in some filter as it is at the moment: Solar energy has grey/black and the same is used for the map basic colors.

So I would suggest to use every color code only once for a specific thing.

**Comment by b2000699** [comment]
Here comes my proposal, maybe it is helpful for you (changes in capitals):
1. Carbon intensity: green-orange-RED
2. Wind power: blue-lila-pink (=As is)
3. Solar power: black-yellow (No white in between)
4. Background: Data available=white, Nodata: Light gray

**Comment by oli** [comment]
@brunolajoie sounds like a good suggestion to me too, but I think you're after @corradio :thinking-face:

:wink:

**Comment by corradio** [comment]
We've already thought extensively about that feature.
The conclusion was that it would clutter the map, and blur the main message we want to carry, which is about carbon intensity, not amount of renewable.
Alternative colouring schemes are interesting for data geeks though. We should probably think of building a premium Electricity Map in order to investigate further correlations and structures.

**Comment by corradio** [comment]
Between green and orange there's yellow, so it won't fix the problem.
Ability to de-activate the carbon intensity is a pretty heavy UX change just to see solar.
I don't have a fix for this right now, but we do keep thinking about it.

**Comment by Francoue** [comment]
Thank you to keep thinking about it and hope you can find a fix. Again, congrats for this remarkable tool !

**Comment by corradio** [comment]
Closing due to inactivity (feel free to re-open if you think I did a mistake closing this!)
